package.json

```diff
@@ -1,4 +1,7 @@
 {
+    "resolutions": {
+      "styled-components": "^5"
```

Comment:
What is this line doing? I think the version is handled below on line 13.

Reply:
I will delete this line